main.py

```diff
@@ -118,10 +117,13 @@ def get_redirect_route(regex_route, handler, defaults=None):
 
 
 def authorization_wrapper(self, *args, **kwargs):
-    # developers.google.com/appengine/docs/python/taskqueue/overview-push
-    # promises that this header cannot be set by external callers. If this
-    # is present, we can be certain that the request is internal and from
-    # the task queue worker.
+    """A request handler to determine if request is internal and from
```

Comment:
Hi @abeerunscore96, I have some suggestions for making this clearer:

```
This request handler wrapper only admits internal requests from taskqueue workers. If the request is invalid, it leads to a 403 Error page.
```

What do you think? I felt it was more succinct and enumerates the different cases clearly. 

In addition, let's put an inline comment below the docstring (just above line 127) along the lines of:

```
# Internal requests should have an "X-AppEngine-TaskName" header (see cloud.google.com/appengine/docs/standard/python/taskqueue/push/).
```

(just to explain what that line is doing, it doesn't really need to be in the top-level docstring because it's an implementation detail). Does that sound good? (Also thanks for fixing the URL!)